src/components/FavoritesList.js

```diff
@@ -59,155 +91,252 @@ const FavoritesList = ({
+												'share/collection/' + list._id + '/' + list.name
+										  )
+										: handleMessageNew(
+												'You must be logged in to share resources'
+										  )
+								}
 								variant="secondary"
 							>
-								<Fa className={classes.marginRight} name="plus" /> Create New
-								List
+								Share
 							</AsylumConnectButton>
-						</div>
+						</Grid>
 					</Grid>
-				)}
-				<Grid container justify="center">
-					<div className={classes.minHeight350}>
-						{loadingResources ? (
-							<Loading />
-						) : (
+				</Grid>
+			)}
+
+			{/* {(session || publicList) && (
+				<Grid
+					container
+					className={classes.container}
+					direction="row"
+					justify="space-between"
+					spacing={0}
+				>
+					{!publicList && (
+						<Grid
+							container
+							className={classes.mainRow}
+							justify="space-between"
+							spacing={0}
+						>
+							
 							<div>
-								{resources.map(
-									(resource) =>
-										resource && (
-											<ResourceListItem
-												isOnFavoritesList={true}
-												isOnPublicList={publicList}
-												handleMessageNew={handleMessageNew}
-												handleListRemoveFavorite={handleRemoveFavorite}
-												history={history}
-												key={resource._id}
-												resource={resource}
-												format="favorites"
-												userData={userData}
-											/>
-										)
+								{list && (
+									<Tooltip
+										className={classes.tooltip + ' hide--on-print'}
+										classes={{tooltipPlacementTop: 'badge-tooltipTop'}}
+										title="Print Favorites"
+										placement="top"
+									>
+										<IconButton
+											color="secondary"
+											style={{height: 'auto'}}
+											onClick={() => {
+												window.print();
+											}}
+										>
+											<Fa name="print" />
+										</IconButton>
+									</Tooltip>
 								)}
+								{list && (
+									<AsylumConnectButton
+										className={classes.marginLeft}
+										onClick={() =>
+											session
+												? handleRequestOpen(
+														'share/collection/' + list._id + '/' + list.name
+												  )
+												: handleMessageNew(
+														'You must be logged in to share resources'
+												  )
+										}
+										variant="secondary"
+									>
+										Share
+									</AsylumConnectButton>
+								)}
+								<AsylumConnectButton
+									className={classes.marginLeft}
+									onClick={() => handleRequestOpen('listNew/favoritesList')}
+									variant="secondary"
+								>
+									<Fa className={classes.marginRight} name="plus" /> Create New
+									List
+								</AsylumConnectButton>
 							</div>
-						)}
-						{!loadingResources && list && resources.length === 0 && (
-							<Typography className={classes.marginTop} variant="body1">
-								You haven't added any resources to this list yet.
-							</Typography>
-						)}
-					</div>
+						</Grid>
```

Comment:
will remove commented code when feature is finished and working as expected. using it to guide the remaining work to complete the feature.